### PR TITLE
03-index-slice-subset.md: a comment on reserved words

### PR DESCRIPTION
**Pull request by jonathanwheeler01**
Some additional information is provided about avoiding naming conflicts with
built-in data types and methods.

Submitting this pull request as part of the checkout process, but am interested in making further changes as needed to improve this contribution.

### CODE REVIEW

File: _episodes/03-index-slice-subset.md [hunk 1]
```diff
@@ -106,6 +106,22 @@ Python tells us what type of error it is in the traceback, at the bottom it says
 `KeyError: 'speciess'` which means that `speciess` is not a valid column name (nor a valid key in
 the related Python data type dictionary).
 
+**NOTE:** The Python language and packages including Pandas define reserved
+words and keywords that should not be used as identifiers when assigning objects
+and variable names. Examples of reserved words in Python include Boolean values
+`True` and `False`, operators `and`, `or`, and `not`, among others. The full list
+of reserved words for Python version 3 are provided at
+[https://docs.python.org/3/reference/lexical_analysis.html#identifiers]
```
Comment (wrightaprilm) [inline _episodes/03-index-slice-subset.md]: extra space between ] and (

File: _episodes/03-index-slice-subset.md [hunk 2]
```diff
@@ -106,6 +106,22 @@ Python tells us what type of error it is in the traceback, at the bottom it says
 `KeyError: 'speciess'` which means that `speciess` is not a valid column name (nor a valid key in
 the related Python data type dictionary).
 
+**NOTE:** The Python language and packages including Pandas define reserved
+words and keywords that should not be used as identifiers when assigning objects
+and variable names. Examples of reserved words in Python include Boolean values
+`True` and `False`, operators `and`, `or`, and `not`, among others. The full list
+of reserved words for Python version 3 are provided at
```
Comment (wrightaprilm) [inline _episodes/03-index-slice-subset.md]: is provided at

**Comment by wrightaprilm** [comment]
Thank you for this! Just a couple small comments.

File: _episodes/03-index-slice-subset.md (hunk 1 above)
Comment (maxim-belkin) [reply _episodes/03-index-slice-subset.md]:
```
<https://docs.python.org/3/reference/lexical_analysis.html#identifiers>
```
produces the same result as
```
[https://docs.python.org/3/reference/lexical_analysis.html#identifiers](https://docs.python.org/3/reference/lexical_analysis.html#identifiers)
```

File: _episodes/03-index-slice-subset.md [hunk 3]
```diff
@@ -106,6 +106,22 @@ Python tells us what type of error it is in the traceback, at the bottom it says
 `KeyError: 'speciess'` which means that `speciess` is not a valid column name (nor a valid key in
 the related Python data type dictionary).
 
+**NOTE:** The Python language and packages including Pandas define reserved
+words and keywords that should not be used as identifiers when assigning objects
```
Comment (maxim-belkin) [inline _episodes/03-index-slice-subset.md]: reserved words = keywords.
no reserved keywords

**Comment by jonathanwheeler01** [comment]
Thank you for your feedback - I have pushed the suggested changes. Happy to make further changes as needed.

File: _episodes/03-index-slice-subset.md [hunk 4]
```diff
@@ -106,6 +106,22 @@ Python tells us what type of error it is in the traceback, at the bottom it says
 `KeyError: 'speciess'` which means that `speciess` is not a valid column name (nor a valid key in
 the related Python data type dictionary).
 
+> ## Reminder
+> The Python language and packages including Pandas define reserved
```
Comment (maxim-belkin) [inline _episodes/03-index-slice-subset.md]: I think it might worth to generalize here:

> The Python language and its modules (such as Pandas) define ...

**Comment by jonathanwheeler01** [comment]
Thank you for the additional feedback - I have made the additional changes, though I was unsure whether to elaborate on the list example by defining and providing a small example of a tuple. Thanks!

**Pull request merged**